test/rtos_drivers/wifi/filesystem_support/wifi_profile.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
-# Copyright (c) 2023, XMOS Ltd, All rights reserved
+# Copyright 2020-2023 XMOS LIMITED.
+# This Software is subject to the terms of the XMOS Public Licence: Version 1.
```

Comment:
Duplicate line.

Reply:
Fixed in https://github.com/xmos/fwk_rtos/pull/219.